lib/web/ui/events.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ui
+
+import "github.com/gravitational/teleport/lib/events"
+
+// EventDescriptions contains short descriptions for events displayed in the UI.
```

Comment:
why having a second lookup table here? this will be problematic in the future, why not just return this in the search api

Reply:
Are we sure we do not want to store event description field as well? It will increase an event footprint but at the same time it also will make things easier. We would just need to store the event and show it as is.

Reply:
I didn't want to save this field in the event as well and didn't really like that I had to have this map and extend it in the api either. I've removed this field. If UI needs to display a short description, it can have it's own map now that there are codes.